### PR TITLE
BaSP-MR-41: Subscriptions Get and Post

### PR DESCRIPTION
**Pull request by jramire5**
Subscription: 

- Added a table showing per each subscription:
  Classes: Day and hour
  Members: Name and last Name, 
  and Date: The date.
- Added a CREATE form that pop ups when clicking a button:
      Showing Classes availables, Members availables and let choose a date

### CODE REVIEW

File: src/Components/Subscriptions/Table/index.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                  className={styles.item}
+                >{`${subscription.classes.day} ${subscription.classes.time}`}</td>
+                {!subscription.members ? (
+                  <td className={styles.item}>{'empty'}</td>
+                ) : (
+                  <td
+                    className={styles.item}
+                  >{`${subscription.members.name} ${subscription.members.lastName}`}</td>
+                )}
+                <td className={styles.item}>{subscription.date}</td>
```
Comment (santimenendez98) [inline src/Components/Subscriptions/Table/index.jsx:29]: This date should show a date in easy format, for example (MM/DD/YYYY)

File: src/Components/Subscriptions/Form-Create/form.module.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: rgba(0, 0, 0, 0.45);
```
Comment (JuanCruzMoreira) [inline src/Components/Subscriptions/Form-Create/form.module.css]: Please keep consistency in color language

File: src/Components/Subscriptions/index.jsx [hunk 3]
```diff
@@ -0,0 +1,119 @@
+import { useEffect, useState } from 'react';
+import styles from './subscriptions.module.css';
+import Table from './Table';
+
+import { CreateModal, ErrorModal } from './Modals';
+import Form from './Form-Create';
+const Subscriptions = () => {
+  const [subscriptions, setSubscriptions] = useState([]);
+  // const [subscriptionById, setSubscriptionById] = useState([]);
```
Comment (JuanCruzMoreira) [inline src/Components/Subscriptions/index.jsx]: Delete commented lines please

**Review by JuanCruzMoreira — CHANGES_REQUESTED** [review]

**Review by JuanCruzMoreira — COMMENTED** [review]
Edir PR title please

File: src/Components/Subscriptions/Form-Create/index.jsx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+      setSubscription({
+        classes: '',
+        members: '',
+        date: ''
+      });
+    } catch (error) {
+      throw new Error(error);
+    }
+  };
+  console.log(subscription.classes);
```
Comment (JuanCruzMoreira) [inline src/Components/Subscriptions/Form-Create/index.jsx]: delete this console.log please

**Review by JuanCruzMoreira — CHANGES_REQUESTED** [review]

**Review by santimenendez98 — APPROVED** [review]

File: src/Components/Subscriptions/Table/index.jsx (hunk 1 above)
Comment (jramire5) [reply src/Components/Subscriptions/Table/index.jsx:29]: Perfect, lets change it when we merge the tables!

Comment (jramire5) [reply src/Components/Subscriptions/Table/index.jsx:29]: So we take it from your table

**Review by JuanCruzMoreira — APPROVED** [review]

**Pull request merged**